paimon-filesystems/paimon-s3/src/main/java/org/apache/paimon/s3/S3Loader.java

```diff
@@ -52,14 +49,6 @@ public String getScheme() {
         return "s3";
     }
 
-    @Override
-    public List<String[]> requiredOptions() {
-        List<String[]> options = new ArrayList<>();
```

Comment:
Can here just check `AWSCredentials awsCredentials = DefaultAWSCredentialsProviderChain.getInstance().getCredentials();`?
If there are awsCredentials in env, return no required option. If there is no awsCredentials, return options.

Reply:
Make sense